src/apps/Permissions/AssignPermissionPanel.js

```diff
@@ -173,8 +173,8 @@ class AssignPermissionPanel extends React.PureComponent {
           </Field>
 
           <Info.Action title="Adding the permission might create a vote">
-            If the permission is created by the Voting app, it will
-            automatically generate a new vote.
+            The Voting app will automatically create a new vote if granting the
+            permission requires a vote to pass.
```

Comment:
Wasn't quite sure what the original text meant, but I think this hasn't changed the intent?

Reply:
Yes that’s better 👍